### PR TITLE
Jedwards/mpiserial update102923

### PR DESCRIPTION
**Pull request by jedwards4b**
Remove the install.patch file and add an install subroutine.
The mpi.mod file was not being included in the original install method.

### CODE REVIEW

**Comment by alalazo** [comment]
@spackbot fix style

**Comment by jedwards4b** [comment]
Just noticed this hasn't been merged yet.

**Comment by alalazo** [comment]
@skosukhin @WardF Would you have any time to review this?

**Comment by skosukhin** [comment]
@alalazo I now wonder what was the reason for my name to appear here :)

**Comment by alalazo** [comment]
@skosukhin You're right, I didn't check the full recipe before. Reconstructed:
- This PR was initially submitted with unwanted changes, involving `netcdf`
- That caused both you and @WardF to be requested as reviewers
- Changes to `netcdf` were undone later.

TL;DR Apologies for the noise, I'll take care of the review :slightly_smiling_face:

**Comment by jedwards4b** [comment]
I'm sorry but why are we still stalled on merging this?

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
> I'm sorry but why are we still stalled on merging this?

Because of turnarounds with PR reviews (apologies, we get hundreds per week, feel free to ping on urgent ones)
